src/main/java/gregtech/api/unification/ore/OrePrefix.java

```diff
@@ -405,6 +417,7 @@ private static void excludeAllGemsButNormal(Material material) {
     private final List<IOreRegistrationHandler> oreProcessingHandlers = new ArrayList<>();
     private final Set<Material> ignoredMaterials = new HashSet<>();
     private final Set<Material> generatedMaterials = new HashSet<>();
+    private final Map<Material, Integer> materialAmounts = new HashMap<>();
```

Comment:
Please use `Object2IntMap<Material> materialAmounts = new Object2IntOpenHashMap<>()`.
Also using `float` instead of `int` might be useful in some cases.

Reply:
i did it for you

Reply:
Thanks !